### DISCUSSION TITLE
Librería de efectos hover para imágenes

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#17] [Announcements]
# Librería de efectos hover para imágenes

Los efectos hover mejoran la experiencia del usuario al poder interactuar con
  el sitio, sobre todo con las imágenes

https://www.zkreations.com/2023/02/libreria-de-efectos-hover-para-imagenes.html

**Comment by danieIabel** [comment]
He actualizado la librería, agregando 8 nuevos efectos.

**Reply by TakoMics** [reply]
Muchas gracias Daniel. Están geniales.

En estos días iniciaré un nuevo proyecto en Blogger, estos efectos me caen buenísimos para el momento.

Un abrazo, saludos.

**Reply by danieIabel** [reply]
Gracias por comentar amigo, espero que le sea de utilidad, seguiré corrigiendo y expandiendo los efectos con el tiempo. Por supuesto yo recomiendo usar solo los efectos que le sean útiles para si no cargar todo el codigo.
